src/panel.js

```diff
@@ -0,0 +1,76 @@
+import Vue from 'vue';
+
+export const PANEL_EMITTER_NAMESPACE = 'instantSearchPanelEmitter';
+export const PANEL_CHANGE_EVENT = 'PANEL_CHANGE_EVENT';
+
+export const createPanelProviderMixin = () => ({
```

Comment:
would be a bit less consistent, but this can just be an object right? Mixins are copied over so it's safe to use the same reference

Reply:
Yep indeed it can, it was only for consistency with the other.